src/components/ValidateCode/ValidateCodeModal.js

```diff
@@ -39,64 +39,52 @@ const defaultProps = {
     },
 };
 
-class ValidateCodeModal extends PureComponent {
-    constructor(props) {
-        super(props);
-
-        this.signInHere = this.signInHere.bind(this);
-    }
-
-    signInHere() {
-        Session.signInWithValidateCode(this.props.accountID, this.props.code);
-    }
```

Comment:
NAB, I think the way we handled this is fine and won't have any bad performance or anything. But wanted to call out that this is a good case for `useCallback()` so instead of inlining the function as an anonymous one we can memoize it like this:

```js
const signInHere = useCallback(() => Session.signInWithValidateCode(props.accountID, props.code), [props.accountID, props.code]);
```

Maybe it's already known here, but just sharing in case not.

Reply:
I like any kind of optimization, even if they are insignificant. Fixed.